admin/routes/check-form-v2.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const express = require('express')
+const router = express.Router()
+const isAuthenticated = require('../authentication/middleware')
+const rolesConfig = require('../roles-config')
+const checkFormV2Controller = require('../controllers/check-form-v2')
+const featureToggles = require('feature-toggles')
+
+if (featureToggles.isFeatureEnabled('newCheckForm')) {
```

Comment:
I don't think we should change to these new urls at all.  Surely there is a better way?

Reply:
@jon-shipley the new urls do not include `v2` anymore as you pointed out. If there's any suggestion for improvements or a better workaround here please let me know.